Lib/test/test_mmap.py

```diff
@@ -266,6 +266,36 @@ def test_access_parameter(self):
                     self.assertRaises(TypeError, m.write_byte, 0)
                     m.close()
 
+    @unittest.skipIf(os.name == 'nt', 'trackfd not present on Windows')
+    def test_trackfd_parameter(self):
+        size = 64
+        with open(TESTFN, "wb") as f:
+            f.write(b"a"*size)
+        with open(TESTFN, "r+b") as f:
+            with mmap.mmap(f.fileno(), size, trackfd=False) as m:
+                self.assertEqual(len(m), size)
+                with self.assertRaises(OSError):
```

Comment:
What is errno?

Reply:
EBADF, directly from the system. I can add it to the test.